ci/build_container/build_container.sh

```diff
@@ -39,59 +41,59 @@ export CC=gcc-4.9
 export CXX=g++-4.9
 
 # libevent
-wget https://github.com/libevent/libevent/releases/download/release-2.1.8-stable/libevent-2.1.8-stable.tar.gz
+wget ${LIBEVENT_HTTP_ARCHIVE}
 tar xf libevent-2.1.8-stable.tar.gz
-cd libevent-2.1.8-stable
-./configure --prefix=$THIRDPARTY_BUILD --enable-shared=no --disable-libevent-regress --disable-openssl
+cd ${LIBEVENT_PREFIX}
+./configure --prefix=${THIRDPARTY_BUILD} --enable-shared=no --disable-libevent-regress --disable-openssl
 make install
 cd ..
 rm -fr libevent*
 
 # BoringSSL
-git clone https://boringssl.googlesource.com/boringssl
+git clone ${BORINGSSL_REMOTE}
 cd boringssl
-git reset --hard b87c80300647c2c0311c1489a104470e099f1531
-cmake .
-make
-cp -r include/* $THIRDPARTY_BUILD/include
-cp ssl/libssl.a $THIRDPARTY_BUILD/lib
-cp crypto/libcrypto.a $THIRDPARTY_BUILD/lib
+git reset --hard ${BORINGSSL_COMMIT}
```

Comment:
Well, for the developer-local build, we were relying on the borginssl BUILD rules that came from the master-with-blaze branch in `bazel/repositories.bzl`. 

We can just invoke a dependency's native build system  in `bazel/repositories.bzl` like we do in`build_container.sh`, but the rules will end up looking like the unholy mess in #680 for its autoconf stuff, since we'll need to copy somewhere sane, do the build, copy out the artifacts and then cleanup. So, it's an option if we can't provide simple BUILD rules ourselves or source them from the dependency's repo, but it's not ideal.